.github/workflows/automated-tests.yml

```diff
@@ -104,6 +104,7 @@ jobs:
         uses: ruby/setup-ruby@v1
         with:
           bundler-cache: true
+          cache-version: 1
```

Comment:
This will [ignore contents of the cache and get and build all the gems anew](https://github.com/ruby/setup-ruby#dealing-with-a-corrupted-cache), and it was added while testing (https://github.com/elzj/otwarchive/commit/64e847e0a05f485a7e2ac6d3632325dbc23d9db9). Unless it's strictly necessary, let's remove it.